pdal/Geometry.cpp

```diff
@@ -187,7 +188,8 @@ SpatialReference Geometry::getSpatialReference() const
     if (srsValid())
     {
         char *buf;
-        m_geom->getSpatialReference()->exportToWkt(&buf);
+        const char *options[] = { "WKT2_2019", nullptr };
```

Comment:
What version of GDAL was `WKT2_2019` introduced, and does this mean PDAL's minimum GDAL version would be limited to that?